.gitignore

```diff
@@ -0,0 +1,2 @@
+node_modules
```

Comment:
There are .gitignore in the backend/ and frontend/ folders already filtering out node_modules, so there is no need for another one here.

Reply:
Removed duplicated .gitignore files and updated the parent file.